depends/packages/native_llvm.mk

```diff
@@ -0,0 +1,34 @@
+package=native_llvm
+$(package)_version=14.0.0-rc2
+ifneq (,$(findstring aarch64,$(BUILD)))
+$(package)_download_path=https://github.com/llvm/llvm-project/releases/download/llvmorg-$($(package)_version)
```

Comment:
This line could moved out of `ifneq` statement,

Reply:
Done in next push.